cpp/0169-majority-element.cpp

```diff
@@ -7,12 +7,16 @@ The majority element is the element that appears more than ⌊n / 2⌋ times. Yo
 class Solution {
 public:
     int majorityElement(vector<int>& nums) {
-        unordered_map<int, int> mp;
-        int n = nums.size();
-        for (int& i : nums){
-           if(++mp[i] > n/2)
-               return i;
-       }
-        return -1;
+        int count{};
+        int res{};
```

Comment:
Could you please replace these with zero initializations to be more clear?

Reply:
This is technically zero initialization (see https://www.learncpp.com/cpp-tutorial/variable-assignment-and-initialization/).

I have replaced it with the proposed zero assignment initialization as requested (if I understand the proposed change correctly).

Reply:
Yes, you are correct, but I believe this is more clear that way as not to get any one confused.